src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let n = 0;
+
+  const calculator = {
+
+    operate(callback, y) {
+      callback.call(this, this.result, y);
+
+      return this;
+    },
+    add(x, y) {
+      n = (x + y);
+      calculator.result = n;
+    },
```

Comment:
Can be less, check other places 
```suggestion
    add(x, y) {
      this.result = x + y;
    },
```